src/app/components/Checkbox/style.ts

```diff
@@ -0,0 +1,30 @@
+import tw, { css } from "twin.macro";
+
+const baseStyle = [tw`form-checkbox h-4 w-4 transition duration-150 ease-in-out`];
+
+const getColor = (color: string): any => {
+	const colorBase = `var(--theme-color-${color})`;
+
+	switch (color) {
+		case "primary":
```

Comment:
The `case "primary":` case could probably be treated as `default:` because primary and default are pretty much the same thing in this context. One thing less to define repeatedly when you use the component as it will be the most common choice.